eq/server/config/resources.cpp

```diff
@@ -312,7 +316,19 @@ bool Resources::discover( ServerPtr server, Config* config,
             name << "display";
         }
         else
+        {
             name << "GPU" << ++gpuCounter;
+#ifdef EQUALIZER_USE_HWSD_gpu_glx
```

Comment:
Why is this GLX-specific? Maybe the detection is only implement in hwsd_glx, but afaik we do not need to restrict it here?